massdriver.yaml

```diff
@@ -133,3 +278,41 @@ ui:
       - enable_cluster
       - shard_count
       - '*'
+  monitoring:
+    ui:order:
+      - mode
+      - alarms
+      - '*'
+    alarms:
+      ui:order:
+        - cpu_metric_alert
+        - memory_metric_alert
+        - server_load_metric_alert
+        - '*'
+      cpu_metric_alert:
+        ui:order:
+          - severity
+          - operator
+          - aggregation
+          - frequency
+          - window_size
+          - threshold
+          - '*'
+      memory_metric_alert:
+        ui:order:
+          - severity
+          - operator
+          - aggregation
+          - frequency
+          - window_size
+          - threshold
+          - '*'
+      server_load_metric_alert:
```

Comment:
Similar to the YAML anchor above, you can define a top level key and << the common order stuff intomplace